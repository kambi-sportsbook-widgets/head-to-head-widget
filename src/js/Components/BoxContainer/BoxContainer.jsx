import React from 'react';
import PropTypes from 'prop-types';
import styles from './BoxContainer.scss';

const BoxContainer = ({ children }) => {
   return (<div className={'KambiWidget-card-inner-border ' + styles.general}>{ children }</div>)
};

BoxContainer.propTypes = {
   children: PropTypes.node
};

export default BoxContainer;
