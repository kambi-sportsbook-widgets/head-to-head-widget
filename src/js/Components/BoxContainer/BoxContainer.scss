:local(.general) {
   padding: 16px 16px;
   max-height: 98px;
}
