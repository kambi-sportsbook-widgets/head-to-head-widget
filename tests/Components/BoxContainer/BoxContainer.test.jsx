/* eslint-env jest */
import React from 'react';
import ReactShallowRenderer from 'react-test-renderer/shallow';
import BoxContainer from '../../../src/js/Components/BoxContainer/BoxContainer';

let renderer;

describe('BoxContainer.render()', () => {

   beforeEach(() => {
      renderer = new ReactShallowRenderer();
   });

   it('renders correctly with no children', () => {
      expect(renderer.render(
         <BoxContainer/>
      )).toMatchSnapshot();
   });

   it('renders correctly with children', () => {
      expect(renderer.render(
         <BoxContainer>
            <div />
            <div />
         </BoxContainer>
      )).toMatchSnapshot();
   });

});
