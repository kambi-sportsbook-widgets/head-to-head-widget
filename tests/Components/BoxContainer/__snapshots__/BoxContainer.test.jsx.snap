// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BoxContainer.render() renders correctly with children 1`] = `
<div
  className="KambiWidget-card-inner-border general"
>
  <div />
  <div />
</div>
`;

exports[`BoxContainer.render() renders correctly with no children 1`] = `
<div
  className="KambiWidget-card-inner-border general"
/>
`;
